src/components/side-bar/index-page-side-bar.js

```diff
@@ -22,9 +20,7 @@ class Anchors extends baseComponents.Anchors {
 
 const SideBarContainer = styled.div`
   position: fixed;
-  top: 50%;
   z-index: 100;
-  transform: translateY(-50%);
```

Comment:
It seems the issue still exists even if we remove vertical center style in some device proportions (e.g. 1280px x 698px).

I came up with another approach:

```js
const SideBarContainer = styled.div`
  /* The fallback goes first */
  position: fixed;
  /* This 'enhancement' is ignored if not understood (e.g. IE 11), */
  /* but overrides the previous value if the browser supports it. */
  position: sticky;
  float: right;
  top: 50vh;
  /*...etc */
`
```

On one hand, we can prevent sidebar from overlapping footer when it supports `position: sticky`. On the other hand, keep the behavior as it is when `position: sticky` is not supported.

## Screenshot
![image](https://user-images.githubusercontent.com/6375655/118118585-b5157400-b41f-11eb-9667-326015c5bd33.png) 

 @achinchen Does this idea also make sense to you ?

Reply:
Further to my previous comment, we still need to remove [this line](https://github.com/twreporter/twreporter-react/blob/3877f99b04a9a7e782dee177effb0985b3acb11a/src/containers/Home.js#L88) with this approach.

Reply:
@taylrj 
Thanks for advice, I made some changes to prevent `float` and be more self-documenting.
Please review the changes, thanks!

Reply:
Thanks for your effort to make it more self-documenting.

However, there is concern that preventing `float` and adding `display: inline-block` result in an inline-level block side bar container which still takes up a whole block.

### screenshot 
![image](https://user-images.githubusercontent.com/6375655/118912560-8c7b0600-b95a-11eb-8619-22ba6c986431.png)

Reply:
@taylrj I try several way to fix the issue, but it looks like `add float` would make thing simple.
Moreover, I tried to tidy up the styles and optimize performance for sticky, I reserved `will-change: transform`
There's the [REF](https://developer.mozilla.org/en-US/docs/Web/CSS/position#performance_accessibility)

Reply:
Thanks for the reference.

🙋‍♂️  Last question, if we add `will-change: transform` to the positioned elements to render the element in its [own layer](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) to improve repaint speed. Why do we need to add `right` as the property value of `will-change` as well ?

Reply:
@taylrj 
Thank you for patiently reviewing.

Due to resize window causes right value to change, though it's very minor. In my opinion, I'd like to add it for optimization.
If there's any concern, let me know, even though removing it isn't a big thing for me.

Reply:
BTW, it's glad to hear you mentioned `stacking context`, that's awesome topic for frontend dev.✨

Reply:
Thanks for the explanation. It's clear.

After a second thought, I think we can follow the suggestion from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change):

> Important: will-change is intended to be used as a last resort, in order to try to deal with existing performance problems. It should not be used to anticipate performance problems.

In view of the suggestion above, `will-change: transform` looks good to me since it helps browser to manage repaints at 60 fps. It is an existing performance issue.
Follow the same suggestion, I prefer to remove `right` from values of `will-change` due to resize window causes value change is a minor case as mentioned in previous comment and it's more like an anticipate performance problem to me.